include/message_filters/subscriber.h

```diff
@@ -156,20 +212,32 @@ class Subscriber : public SubscriberBase, public SimpleFilter<M>
    *
    * \param node The rclcpp::Node to use to subscribe.
    * \param topic The topic to subscribe to.
-   * \param qos (optional) The rmw qos profile to use to subscribe
+   * \param qos (optional) The qos profile to use to subscribe
    */
-  // TODO(wjwwood): deprecate in favor of API's that use `rclcpp::QoS` instead.
```

Comment:
In order to remove this, I'd recommend actually deprecating the old version in favor of the new one that requires the QoS be specified.

Reply:
Yes, you are right. I would have removed the old API, but I suppose there is much code depending on this.  I have added a message to the rmw methods: ` [[deprecated("Use rclcpp::QoS to specify qos")]]`